.github/workflows/live-deploy.yml

```diff
@@ -0,0 +1,30 @@
+# An Action to build Web Arcs and deploy to https://live.arcs.dev
```

Comment:
Is `.github/workflows/live-deploy.yml` the right directory? Or, should this move to tools?

Reply:
No, it needs to live in this directory. `.github` is a reserved folder for actions + github automation.